.github/workflows/run-k8s-tests-on-ppc64le.yaml

```
@@ -0,0 +1,81 @@
+name: CI | Run kubernetes tests on Power(ppc64le)
+on:
+  workflow_call:
+    inputs:
+      registry:
+        required: true
+        type: string
+      repo:
+        required: true
+        type: string
+      tag:
+        required: true
+        type: string
+      pr-number:
+        required: true
+        type: string
+      commit-hash:
+        required: false
+        type: string
+      target-branch:
+        required: false
+        type: string
+        default: ""
+
+jobs:
+  run-k8s-tests:
+    strategy:
+      fail-fast: false
+      matrix:
+        vmm:
+          - qemu
+        k8s:
+          - kubeadm
+    runs-on: ppc64le
+    env:
+      DOCKER_REGISTRY: ${{ inputs.registry }}
+      DOCKER_REPO: amulyam24
```

Was this just hard-coded for testing and should be switched back now?
```suggestion
      DOCKER_REPO: ${{ inputs.repo }}
```

---

That's right, I changed it while testing and missed correcting it. Rectified it.